fb_apt/README.md

```diff
@@ -0,0 +1,88 @@
+fb_apt Cookbook
+====================
+The `fb_apt` cookbook installs and configures APT, the Debian package 
+management tool.
+
+Requirements
+------------
+
+Attributes
+----------
+* node['fb_apt']['config']
+* node['fb_apt']['keys']
+* node['fb_apt']['keyring']
+* node['fb_apt']['keyserver']
+* node['fb_apt']['mirror']
+* node['fb_apt']['preserve_sources_list_d']
+* node['fb_apt']['preferences']
+* node['fb_apt']['repos']
+* node['fb_apt']['update_delay']
+* node['fb_apt']['want_backports']
+* node['fb_apt']['want_non_free']
+* node['fb_apt']['want_source']
+
+Usage
+-----
+To install and configure APT include `fb_apt`, which will populate the 
+repository sources in `/etc/apt/sources.list` and update the package cache 
+every `node['fb_apt']['update_delay']` seconds (defaults to 86400) using the 
+`execute[apt-get update]` resource (which other cookbooks can suscribe to or 
+notify against). Note that the `node['fb_apt']['update_delay']` attribute is 
+evaluated at compile time, so you'd need to set it before including `fb_apt` if
+you'd like to customize it.
+
+### Repository sources
+By default the cookbook will setup the base distribution repos based on the
+codename (as defined in `node['lsb']['codename']`). The mirror can be
+customized with `node['fb_apt']['mirror']`; if set to `nil`, base repos will
```

Comment:
no I think you misunderstand. I'm saying that if the person sets the mirror to 'nil'...  mkaing their system unusable seems bad... especially because by defualt we also blow-away sources.d

But given your default isn't nil, it makes sense. UPdate to say that the default is a global mirror for wahtever your distro is